app/api/search/route.ts

```diff
@@ -1,11 +1,20 @@
 import { getPages } from '@/utils/docs-loader';
```

Comment:
Let's refactor the docs-loader to export { getDocsPage, getDocsPages, docsPageTree }